pubspec.yaml

```diff
@@ -9,6 +9,8 @@ environment:
 dependencies:
   analyzer: ">=0.30.0 <0.31.0"
   barback: "^0.15.0"
+  built_redux: ^6.1.0
```

Comment:
This dependency worries me since it's undergoing major version bumps quite frequently. If we include this, that means we have to stay on top of bumping the upper bound to avoid being a source of version lock.

Reply:
Yeah, that's a good point

Reply:
It only looks that way because, per @davidmarne-wf, because he went 1.0 too early and broke some things so he had to rev for semver purposes.

Reply:
Yeah, I understand your hesitation for sure, but I really do believe it finally settled down. I'm very happy with where it stands right now.

Plus at this point it is a dependency for CDS, w_comments, review_bar, and soon to be graph_app (and many other repos via transitivity from w_comments) so any major bumps are already going to have to be dealt with anyways. 

However, if we want I could easy make another repo that just has this component.